apprise/py3compat/asyncio.py

```diff
@@ -36,25 +36,6 @@
     (sys.version_info.major == 3 and sys.version_info.minor >= 7)
```

Comment:
This is amazing work @YoRyan .

Do you think we even need the `asyncio.py` file anymore?  The class defiend here was also just a workaround to Python v2.7.  Or is it useful to keep around?

Reply:
Ooh, I like your thinking; it should be quite possible to remove, yeah. The AsyncNotifyBase subclass can be merged into NotifyBase. As for the tosync() function, it seems to be using two different techniques for Python 3.6 and 3.7+. There might be way to combine, or at least significantly simplify those branches. I'll have to investigate.

Just to confirm, we now only support 3.6 and above, right?

Reply:
> Just to confirm, we now only support 3.6 and above, right?

Yes, that's right.  Just Python v3.6+ :+1: 

> The AsyncNotifyBase subclass can be merged into NotifyBase.

That's what i had in mind; likely just drop the `AsyncNotifyBase` in entirety.  I think there are just a few functions that need to move over... maybe even worth double underscoring them so `function __internal actions` since they shouldn't ever be called directly anyway.. just `notify()`, and `await async_notify()`.

The only other thing i'd like to try to preserve is the variable set in the `AppriseAsset()` object.  Would be great if we can still set `async = False` when calling `notify()` (non asynchronous version) and fall back to the way it is today and sequentially call one after the other.  I don't think keeping this backwards compatibility should amount to too much extra logic lingering behind.

Reply:
Both should be accomplished with the latest commit. I've eliminated the subclass, and if you set async_mode=False you get a synchronous loop through each send() call, the old-fashioned way. 👍